spec/player_o_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'player_o'
+
+describe PlayerO do
+
+  let(:game) { double :game, board: Board.new }
+  subject(:player_o) { described_class.new(game.board) }
```

Comment:
Encapsulation: Should the game object allow access to its inner state (the board)? Try Googling "Tell, don't ask" for more info.